vendor/engines/saas/app/jobs/saas/trials_create_job.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
+  class TrialsCreateJob < ActiveJob::Base
+
+    def perform(params)
+      user = params['current_user']['login']
+      params['url'] = "/settings/#{user}/trialed"
+      Analytics::PageJob.perform_later(params)
+    end
```

Comment:
In this case if `current_user` is nil I would hope for an exception to help me trace where trial sign-ups are going wrong. Seem reasonable ?